config/default.js

```diff
@@ -429,4 +429,13 @@ module.exports = {
   // assumption is if LOCAL_PROPOSER is true, wallet UI app
   // is running in local machine
   isLocalRun: process.env.LOCAL_PROPOSER === 'true',
+  BLOCK_TYPES: '(uint48,address,bytes32,uint256,bytes32,bytes32)',
```

Comment:
for organizational sake, can we make these types into a nested object? like:

```
{
...,
"TYPES": {
    "BLOCK": '(uint48,address,bytes32,uint256,bytes32,bytes32)',
    etc
},
...
}
```

Reply:
To be specific to their exact function of function/structure signatures, `SIGNATURES` would be very apt

Reply:
done